.travis.yml

```diff
@@ -12,10 +12,10 @@ install:
   - make bootstrap
 before_script:
   # Download kubectl, which is a requirement for using minikube.
-  - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.8.0/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+  - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.9.4/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
```

Comment:
Let us provide comments on why we are using this particular version. Can we provide details on the other versions supported by openebs. A contributor might believe that this is the k8s version that openebs supports?

Reply:
openebs volume delete is experiencing an issue with k8s 1.9.2
Has the integration tests run before submitting this? Can IT run be run as travis build on this branch?

Reply:
@amitd This is the fix cherry picked from kiran's fix and merge on 0.5 branch. https://github.com/openebs/maya/pull/303